src/brain/issueLabelHandler/index.test.ts

```diff
@@ -738,7 +773,7 @@ describe('issueLabelHandler', function () {
     it('should not modify labels when community member comments and issue is waiting for product owner', async function () {
       await setupIssue();
       await addLabel(WAITING_FOR_PRODUCT_OWNER_LABEL, 'routing-repo');
-      await addComment('routing-repo', 'Picard');
+      await addComment('routing-repo', 'Skywalker');
```

Comment:
How does this test still pass if we change the user adding the comment to the opposite of before?

Also, mixing Star Wars and Star Trek references is sacrilege!

Reply:
Before, the functionality that existed was the same for when a user with either `OWNER` or `COLLABORATOR` permissions commented on an issue that was `Waiting for: Product Owner` (nothing should happen). Now, when an `OWNER` comments on an issue that is `Waiting for: Product Owner`, the label should be removed. That's why this test case was failing, since there was a typo